pkg/controllers/cloud/networkpolicy_controller.go

```diff
@@ -62,6 +61,12 @@ const (
 	npSyncReadyBookMarkCnt = 3
 )
 
+var (
+	cloudSyncInterval   int64
```

Comment:
move this inside NetworkPolicyReconciler

Reply:
Solved.